src/h3/_cy/h3lib.pxd

```diff
@@ -149,13 +141,28 @@ cdef extern from "h3api.h":
     void getH3UnidirectionalEdgeBoundary(H3Index edge, GeoBoundary *gb)
 
     int h3LineSize(H3Index start, H3Index end)
-
     int h3Line(H3Index start, H3Index end, H3Index *out)
 
     int maxFaceCount(H3Index h3)
-
     void h3GetFaces(H3Index h3, int *out)
 
     int experimentalH3ToLocalIj(H3Index origin, H3Index h3, CoordIJ *out)
-
     int experimentalLocalIjToH3(H3Index origin, const CoordIJ *ij, H3Index *out)
+
+    double hexAreaKm2(int res) nogil
+    double hexAreaM2(int res) nogil
```

Comment:
It occurs to me after adding this in Java that the existing hexArea functions don't have a radians or radians^2 unit option in the C library.

Reply:
That's true, but I figured they would be useful for folks if they were looking at, say, other planets with a different radius.

Should we add a `hexAreaRads2`?